src/components/CirclePagination/CirclePagination.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import { Row, Box } from "../../theme/custom-styled-components";
+import { ColorType } from "../../theme/util";
+import Circle from "./Circle";
+
+export interface CirclePaginationProps {
+  activeItem: number;
+  data: any;
+  bgColor?: ColorType;
+  onItemClick: (index: any) => void;
+  className?: string;
+}
+
+const ButtonStyled = styled(Box)`
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+`;
+
+export const CirclePagination: FC<CirclePaginationProps> = ({
+  onItemClick,
+  data,
+  bgColor,
+  activeItem,
+  className,
+}: CirclePaginationProps) => {
+  return (
+    <Row className={className} mt={{ xxs: 3, lg: 5 }}>
```

Comment:
All the components in the pattern library should be self-contained so we shouldn't add any "outer spacing" within the component. This will allow for more flexibility. Remove margin-top styling here and apply the margin using the custom  <Spacer/>` styled-system component in `CirclePagination.stories.tsx`.

Future reading about [handling spacing in a UI component library](https://medium.com/fed-or-dead/handling-spacing-in-a-ui-component-library-70f3b22ec89).